gateway-service-definitions/src/main/resources/services/livy/0.4.0/service.xml

```diff
@@ -24,12 +24,8 @@
     </metadata>
   <routes>
     <route path="/livy/**?**"/>
-    <route path="/livy">
```

Comment:
Like mentioned previously we need an entry to call the filter.

Reply:
Sure Added the relevant change.